wire-ios-request-strategy/Sources/Helpers/EncryptionSessionDirectory+UpdateEvents.swift

```diff
@@ -82,8 +83,7 @@ extension EncryptionSessionsDirectory {
 
     /// Appends a system message for a failed decryption
     fileprivate func appendFailedToDecryptMessage(after error: CBoxResult?, for event: ZMUpdateEvent, sender: UserClient, in moc: NSManagedObjectContext) {
-        zmLog.safePublic("Failed to decrypt message with error: \(error), client id <\(sender.safeRemoteIdentifier))>")
-        zmLog.error("event debug: \(event.debugInformation)")
+        logger.error("Failed to decrypt message with error: \(error.debugDescription), client id <\(sender.safeRemoteIdentifier))>")
```

Comment:
I would actually add this in addition to the `zmLog`, because they're logs that appear in different places.